src/main/java/org/broadinstitute/consent/http/service/TDRService.java

```diff
@@ -0,0 +1,32 @@
+package org.broadinstitute.consent.http.service;
+
+import com.google.inject.Inject;
+import org.broadinstitute.consent.http.models.Dataset;
+import org.broadinstitute.consent.http.models.User;
+import org.broadinstitute.consent.http.models.tdr.ApprovedUser;
+import org.broadinstitute.consent.http.models.tdr.ApprovedUsers;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TDRService {
+    private final DataAccessRequestService dataAccessRequestService;
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    @Inject
+    public TDRService(DataAccessRequestService dataAccessRequestService) {
+        this.dataAccessRequestService = dataAccessRequestService;
+    }
+
+    public ApprovedUsers getApprovedUsersForDataset(Dataset dataset) {
+        Collection<User> users = this.dataAccessRequestService.getUsersApprovedForDataset(dataset);
+
+        List<ApprovedUser> approvedUsers =
+                users.stream().map((u) -> new ApprovedUser(u.getEmail())).collect(Collectors.toList());
```

Comment:
Minor suggestion. We could sort the list so the response is consistent across API calls. JSON doesn't enforce order, but we can make that happen if we want.